src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,58 @@
+# Common data types
+
+## Preprocessed or cleaned data
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Preprocessing in this context means transformations of data that do not change
+the data type of the input (as expressed by its BIDS `suffix`).
+For instance, a change in the number of dimensions is likely to disrupt the propagation
+of the input's `suffix` and generally, the outcomes of such transformation
+cannot be considered _preprocessed_ or _cleaned data_.
+Examples:
+
+ -  Motion-corrected, temporally denoised, and transformed to MNI space BOLD series
+ -  Inhomogeneity corrected and skull stripped T1w files
+ -  Motion-corrected DWI files
+ -  Time-domain filtered EEG data
+ -  Spatially filtered EEG data
+
+The `space` keyword is recomended to distinguish files with different underlying
+coordinate systems or registered to different reference maps.
+The `desc` (description) keyword is a general purpose field with freeform values,
+which SHOULD be used to distinguish between multiple different versions of
+processing for the same input data.
+
+Note that even though `space` and `desc` are optional at least one of them MUST
+be defined to avoid name conflict with the raw file.
```

Comment:
This MUST is not consistent with the more general [File naming conventions](https://bids-specification.readthedocs.io/en/common-derivatives/05-derivatives/01-introduction.html#file-naming-conventions) which states that a requirement is that the file name is different (from raw filenames), which can be either in the suffix, and/or in derivative specific keywords.

I believe what is meant here is that a filename MUST be unique, and that (in this context) it could be `space` or `desc` keyword.

I recommend that this sentence on file naming is removed here, and that specifications of general naming requirements are restricted to the more elaborate "File naming conventions" section in the introduction.